sql/schema.sql

```diff
@@ -15,7 +15,9 @@ DROP TABLE IF EXISTS `ghtorrent`.`users` ;
 CREATE TABLE IF NOT EXISTS `ghtorrent`.`users` (
   `id` INT(11) NOT NULL AUTO_INCREMENT COMMENT '',
   `login` VARCHAR(255) NOT NULL COMMENT '',
+  `name` VARCHAR(255) NULL DEFAULT NULL COMMENT '',
```

Comment:
This (and the `email`) are not fields that are distributed with the default MySQL dump (which is what this file is meant to support), due to ghtorrent/ghtorrent.org#32